tests/unit_tests/db_engine_specs/test_gsheets.py

```diff
@@ -159,44 +139,33 @@ def test_validate_parameters_catalog_and_credentials(
 
     parameters: GSheetsParametersType = {
         "credentials_info": {},
-        "query": {},
-        "table_catalog": {
+        "catalog": {
             "private_sheet": "https://docs.google.com/spreadsheets/d/1/edit",
             "public_sheet": "https://docs.google.com/spreadsheets/d/1/edit#gid=1",
             "not_a_sheet": "https://www.google.com/",
         },
     }
-    errors = GSheetsEngineSpec.validate_parameters(parameters)
+    errors = GSheetsEngineSpec.validate_parameters(parameters)  # ignore: type
+
     assert errors == [
         SupersetError(
-            message=(
-                "Unable to connect to spreadsheet not_a_sheet at "
-                "https://www.google.com/"
-            ),
+            message="Unable to connect to spreadsheet not_a_sheet at https://www.google.com/",
             error_type=SupersetErrorType.TABLE_DOES_NOT_EXIST_ERROR,
             level=ErrorLevel.WARNING,
             extra={
+                "invalid": ["catalog"],
                 "name": "not_a_sheet",
                 "url": "https://www.google.com/",
                 "issue_codes": [
                     {
                         "code": 1003,
-                        "message": (
-                            "Issue 1003 - There is a syntax error in the SQL query. "
-                            "Perhaps there was a misspelling or a typo."
-                        ),
+                        "message": "Issue 1003 - There is a syntax error in the SQL query. Perhaps there was a misspelling or a typo.",
                     },
                     {
                         "code": 1005,
-                        "message": (
-                            "Issue 1005 - The table was deleted or renamed in the "
-                            "database.",
-                        ),
+                        "message": "Issue 1005 - The table was deleted or renamed in the database.",
                     },
                 ],
             },
-        ),
+        )
     ]
-    create_engine.assert_called_with(
-        "gsheets://", service_account_info={}, subject="admin@example.com",
-    )
```

Comment:
Why are we getting rid of this test?

Reply:
no clue how these got removed guess it was a bad merge conflict on my end